app/screens/channel/header/bookmarks.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+
+import ChannelBookmarks from '@components/channel_bookmarks';
+import {useTheme} from '@context/theme';
+import {useDefaultHeaderHeight} from '@hooks/header';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    canAddBookmarks: boolean;
+    channelId: string;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    container: {
+        backgroundColor: theme.sidebarBg,
+        width: '100%',
+        position: 'absolute',
+    },
+    content: {
+        backgroundColor: theme.centerChannelBg,
+        borderTopLeftRadius: 12,
+        borderTopRightRadius: 12,
+    },
+    separator: {
+        height: 1,
+        backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+    },
+    separatorContainer: {
+        backgroundColor: theme.centerChannelBg,
+        zIndex: 1,
+    },
+    padding: {
+        paddingTop: 2,
```

Comment:
Do we need this? It's adding height to the bookmarks bar. I removed it locally and seemed ok to remove